functests/utils/nodes/nodes.go

```diff
@@ -21,6 +22,13 @@ import (
 	testlog "github.com/openshift-kni/performance-addon-operators/functests/utils/log"
 	"github.com/openshift-kni/performance-addon-operators/pkg/controller/performanceprofile/components"
 )
+//NumaNodes defines cpus in each numa node
+type NumaNodes struct {
+	Cpus []struct {
```

Comment:
Thanks for the comments, I will have to use string because the lscpu command gives the json output in strings:

```
(venv) [mniranja@mniranja]$ lscpu -e=node,cpu -J
{
   "cpus": [
      {"node": "0", "cpu": "0"},
      {"node": "0", "cpu": "1"},
      {"node": "0", "cpu": "2"},
      {"node": "0", "cpu": "3"},
      {"node": "0", "cpu": "4"},
      {"node": "0", "cpu": "5"},
      {"node": "0", "cpu": "6"},
      {"node": "0", "cpu": "7"}
   ]
}
```
I will modify the structure and update the patch.